docs/testnet/onomy-testnet-docs/faucet.md

```diff
@@ -42,44 +43,74 @@ Note 'myvalidatorkeyname' is just the name of your key here, you can pick anythi
 You'll be prompted to create a password, I suggest you pick something short since you'll be typing it a lot
 
 ```
-cd $HOME
-onomyd --home $HOME/onomy/onomy init mymoniker --chain-id onomy
-onomyd --home $HOME/onomy/onomy keys add myvalidatorkeyname --keyring-backend test --output json | jq . >> $HOME/onomy/onomy/validator_key.json
+onomyd --home $HOME/onomy-testnet1/onomy keys add {myvalidatorkeyname} --keyring-backend test --output json | jq . >> $HOME/onomy-testnet1/onomy/validator_key.json
 ```
 
 ### Copy the genesis file
 
 ```
-rm $HOME/onomy/onomy/config/genesis.json
-wget http://147.182.128.38:26657/genesis? -O $HOME/raw.json
-jq .result.genesis $HOME/raw.json >> $HOME/onomy/onomy/config/genesis.json
+rm $HOME/onomy-testnet1/onomy/config/genesis.json
+wget http://147.182.190.16:26657/genesis? -O $HOME/raw.json
```

Comment:
As I see you nad to update the docs, because of the change of the IP. So maybe we could you DNS?

Reply:
once DNS testing done, accordingly will update ip to DNS url